Task_DEV-4/Task_DEV-4/Materials.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Text;
+using System;
+using System.IO;
+
+namespace Task_DEV_4
+{
+    /// <summary>
+    /// class including description about entity
+    /// </summary>
+    class Materials : Id
+    {
+        private string textInformation { get; set; }
+        public string TextInformation
+        {
+            get
+            {
+                return textInformation;
```

Comment:
If the body of the get or set method of a property consists of a single statement, the statement is
written on the same line as the method signature. White space is inserted between the property
method (get, set) and the opening brace. This will create visually more compact class definitions..